src/client/pages/OfferNew/utils.ts

```diff
@@ -337,6 +337,7 @@ export const typeOfResidenceTextKeys: Record<
   [TypeOfContract.SeApartmentStudentBrf]:
     'CHECKOUT_DETAILS_RESIDENCE_TYPE_OWN_APARTMENT',
   [TypeOfContract.SeHouse]: 'CHECKOUT_DETAILS_RESIDENCE_TYPE_OWN_HOUSE',
+  [TypeOfContract.NoHouse]: 'CHECKOUT_DETAILS_RESIDENCE_TYPE_OWN_HOUSE',
```

Comment:
Why did we add NoHouse to start with?

Reply:
in order to pass typcheck. it's the same as the adtraction discussion we had on slack yesterday.